server/index.js

```diff
@@ -171,7 +170,7 @@ module.exports = (oldConfig = {}) => {
         ? config.canvasHost
         : req.session.canvasHost || config.canvasHost
     );
-    const accessToken = (req.session.accessToken || config.accessToken);
+    const accessToken = (req.accessToken || config.accessToken);
```

Comment:
I'm assuming this will make sense when i get to some of the other PRs

Reply:
I hope so!